packages/models/src/inscriptions/inscriptions.model.ts

```diff
@@ -0,0 +1,21 @@
+export interface BestinslotInscriptionResponse {
```

Comment:
I think we were making an effort to keep api models with the queries and only use the models pkg for the models we map/create/use in the app ...like a DTO pattern?

Reply:
+1, this is specific to the API so belongs with the fetcher/query. 

`@leather.io/models` shouldn't "know" about BIS.